### ISSUE TITLE
User icon wrong shape in Safari

### ISSUE THREAD

**Issue by dbkr** [element-hq/element-call#640]
### Steps to reproduce

1. Open EC in Safari on mac
2. Click on the user icon in the top right to open the menu
3. Click somewhere else to close the menu

#### What did you expect?
Normal round user icon

#### What happened instead?
![Screenshot 2022-10-19 at 16 27 30](https://user-images.githubusercontent.com/986903/196735822-83305f0e-10c9-4ef6-9098-5845c4eb0fa7.png)

### Operating system

macOS

### Browser information

Safari 16.0

### Will you send logs?

No

**Comment by daniellekirkwood** [comment]
i don't think we have this issue anymore, i'm going to close the ticket but please reopen if i'm wrong!
